### DISCUSSION TITLE
Responsible disclosure policy

### DISCUSSION THREAD

**Discussion by erdnaxe** [mat/besticon#88] [General]
Currently there is no methods to disclose a potential security issues. Could you add a SECURITY.md file? GitHub [recommends](https://docs.github.com/en/code-security/getting-started/adding-a-security-policy-to-your-repository) a security policy to ensure issues are responsibly disclosed.

Looking forward to hearing from you!

**Comment by mat** [comment]
Thanks for the suggestion! https://github.com/mat/besticon/blob/master/.github/SECURITY.md
